api/swagger/swagger.json

```diff
@@ -9824,6 +9819,11 @@
                     }
                   }
                 },
+                "node_id": {
+                  "description": "Node identifier on which a service is been running.\nExactly one of these parameters should be present: node_id, node_name, add_node.",
+                  "type": "string",
+                  "x-order": 0
+                },
```

Comment:
@BupycHuk May you take a look:
* why this definition jumped there?
* why `add_node` does not have `x-order` at all?

Reply:
ok, I'll take a look

Reply:
it's because swagger flatten command replaces whole block.